dashboard/app/models/levels/bubble_choice.rb

```diff
@@ -141,8 +141,16 @@ def summarize_sublevels(script_level: nil, user_id: nil, should_localize: false)
         level_url(level.id)
 
       if user_id
-        level_info[:perfect] = UserLevel.find_by(level: level, user_id: user_id)&.perfect?
-        level_info[:status] = level_info[:perfect] ? SharedConstants::LEVEL_STATUS.perfect : SharedConstants::LEVEL_STATUS.not_tried
+        level_info[:perfect] = UserLevel.find_by(
+          level: level,
+          script: script_level.try(:script),
+          user_id: user_id
+          )&.perfect?
+        level_info[:status] = if level_info[:perfect]
```

Comment:
Mostly curious, why this change?  The ternary seems more canonical and readable?

Reply:
we have a [linting rule](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MultilineTernaryOperator) that disallows multiline ternaries, and while we don't have a linting rule for line length, i still like to respect our style guide line length in any code i touch. i agree that the ternary seems more readable, but apparently that's not "the ruby way" to do it!